.github/workflows/ci.yml

```diff
@@ -6,10 +6,11 @@ jobs:
   test:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - uses: php-actions/composer@v6
       - name: PHPUnit Tests
         uses: php-actions/phpunit@v3
         with:
           configuration: phpunit.xml
-          php_version: 8.2
+          bootstrap: vendor/autoload.php
+          test_suffix: Test.php
```

Comment:
This test suffix is what was breaking the pipeline. Its original value in the phpunit action is set to `Test.php, .phpt` which breaks it since phpunit no longer supports the separator in there which is why I set it to just `Test.php` for now.